src/magic_folder/snapshot.py

```diff
@@ -216,35 +216,39 @@ def create_author_from_json(data):
     return create_author(data["name"], verify_key)
 
 
-def sign_snapshot(local_author, snapshot, content_capability):
+def sign_snapshot(local_author, snapshot, content_capability, metadata_capability):
     """
     Signs the given snapshot with provided author
 
     :param LocalAuthor local_author: the author to sign the data with
 
     :param LocalSnapshot snapshot: snapshot to sign
 
-    :param str content_capability: the Tahoe immutable
+    :param bytes content_capability: the Tahoe immutable
         capability-string of the actual snapshot data.
 
+    :param bytes metadata capability: the Tahoe immutable
+        capability-string of the metadata (which is serialized JSON)
+
     :returns: bytes representing the signature (or exception on
         error).
     """
     # XXX See
     # https://github.com/LeastAuthority/magic-folder/issues/190 as
-    # this is likely insufficient (and hasn't been looked at by our
-    # cryptograhphers yet).
+    # this hasn't been looked at by our cryptograhphers yet).
     data_to_sign = (
         u"{content_capability}\n"
+        u"{metadata_capability}\n"
         u"{name}\n"
     ).format(
         content_capability=content_capability,
```

Comment:
Formatting bytes into unicode might not be ideal.  The capabilities are bytes, per the docstring.  The path is unicode, I think.  Maybe encoding the path first and then interpolating into bytes is better?

The only practice difference this might make is if the capabilities have non-ascii in them somehow, then the interpolation will fail with a `UnicodeDecodeError`, I think.